build.gradle

```diff
@@ -7,13 +7,10 @@ buildscript {
     repositories {
         jcenter()
         mavenLocal()
-        maven {
-            url 'https://maven.google.com/'
-            name 'Google'
-        }
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.1.3'
```

Comment:
Maybe 3.1.4.

Reply:
Changing compile to something else is breaking ReactAndroid build, and think that it needs some refactoring. IIRC, 3.1.3 is the latest Android Plugin for Gradle.

Reply:
IIRC, 3.1.3 is the latest version.